_minutes/2021-10-28-wecg.md

```diff
@@ -0,0 +1,120 @@
+ * [timothy] Think there are minimal differences between MV2 and MV3 manifests. Safari is very relaxed when it comes to erroring for unknown keys
+ * [carlos] users are often confused about what is recognised or not.
+
+[Issue #109](https://github.com/w3c/webextensions/issues/109): Use cases not covered by DNR: POST payloads
+
+ * [giorgio] possible with blocking WR, not with DNR. Preference for Firefox style WebRequest as it's non-blocking.
+ * [tomislav] do you know a specific use case you need to address about POST request?
+ * [giorgio] XSS filter – need to be able to analyze the POST request. Sites can work around content blocking by switching to POST
+ * [timothy] Can we work around this with declarative rules?
+ * [giorgio] Probably need to be a callback, based on heuristic, transform the data, perform pattern matching for XSS payloads.
+ * [simeon] open to the possibility of logic execution, problem with applying function in browser environment, leaking info. How to tackle arbitrary code execution.  Main problem is amount of data that extensions has access to.
+ * [krzysztof] other APIs expose a similar level of access, like content scripts, cookies.
+ * [tomislav] FF team is considering dNR a fallback that will be something similar to
+ * [simeon] Chrome teams MV3 initial motivators was wanting to change relation to broad host permissions (&lt;all_urls>), we plan to change &lt;all_urls> permissions to be opt-in by the user, temporary permission grants in response to user actions.  DNR designed to address that usecase without exposing all the info to extensions.
+ * [timothy] broadly in agreement about broad permissions.
+ * [tomislav] we're working on similar user access controls for permissions, but once granted, there are usecases that can't be declarative.
+ * [giorgio] want to understand the direction we're going. Are we going to have to see permissions prompts on every web page
+ * [timoty] added “run on all websites” for common usecases like tab managers , password managers
+ * [simeon] working on UI for a while, not ready yet.  Napkin sketch is all we could commit a few months ago.  Long term, we want to go to a model of only acting based on users actions.  Don't think we can get rid of &lt;all_urls>, but want to move to a more reactive model.  One of the solutions is “in-context-badging”, text boxes on a page, input boxes on a page flagged with grammar extension/shopping assistant.
+ * [tomislav] broadly in agreement with the goals and direction. Noscript is an example of something that requires all_urls.
+ * [giorgio] understand that browsers want to restrict access to the narrowest set of permissions, think the judgement on this could be handled by extension store reviewers
+ * [carlos] #199 optional_host_permissions not all permissions need to be asked at installation time, some might be required, but others truly optional.
```

Comment:
```suggestion
 * [carlos] [#119](https://github.com/w3c/webextensions/issues/119) optional_host_permissions not all permissions need to be asked at installation time, some might be required, but others truly optional.
```

Wrong bug reference.

Also, during minute-taking I also linkify issue references, which would result in links in the output here.

I've updated the source doc with this.